polygeist/tools/cgeist/Test/Verification/use-global-in-namespace.cpp

```diff
@@ -0,0 +1,18 @@
+// RUN: clang++ -fsycl -fsycl-targets=spir64-unknown-unknown-syclmlir -emit-mlir -O0 %s -o - -S -emit-mlir -fsycl-device-only | FileCheck %s
+
+#include <sycl/sycl.hpp>
+
+namespace NS {
+// CHECK-LABEL:     llvm.mlir.global internal @_ZN2NSL1CE() {addr_space = 0 : i32, sym_visibility = "private"} : f64 {
```

Comment:
How does this test check that the global is correctly placed in the device module?

Reply:
Compilation would fail otherwise as "host" module is cleaned first-thing after codegen.

Reply:
I'll add an explicit check anyway. Will rebase on `sycl` first...